cluster/saltbase/salt/kube-scheduler/kube-scheduler.manifest

```diff
@@ -8,7 +8,10 @@
 {
 "apiVersion": "v1",
 "kind": "Pod",
-"metadata": {"name":"kube-scheduler"},
+"metadata": {
+  "name":"kube-scheduler",
+  "namespace": "kube-system"
```

Comment:
Are we sure we want all master components are in the same namespaces as our addons?